packages/shared/src/buttons/ButtonSecondary.js

```diff
@@ -0,0 +1,57 @@
+// @flow strict
```

Comment:
This differs from button only by backgroundColor right?
Can we rather reuse the button component and pass it type="secondary" or something like that?